### PR TITLE
create repo for kotlin sdk

### PR DESCRIPTION
**Pull request by MP91**

### CODE REVIEW

File: otterdog/eclipse-velocitas.jsonnet [hunk 1]
```diff
@@ -317,6 +317,25 @@ orgs.newOrg('eclipse-velocitas') {
...
       ],
     },
+    orgs.newRepo('vehicle-app-kotlin-sdk') {
+      allow_update_branch: false,
+      description: "Vehicle App sdk for Kotlin",
+      has_wiki: false,
+      is_template: false,
+      web_commit_signoff_required: false,
+      workflows+: {
+        default_workflow_permissions: "write",
```
Comment (netomi) [inline otterdog/eclipse-velocitas.jsonnet:327]: I would strongly suggest to use read permissions by default.

Comment (MP91) [reply otterdog/eclipse-velocitas.jsonnet:327]: just copied the settings from the other repositories, so all have the same setup...

Comment (netomi) [reply otterdog/eclipse-velocitas.jsonnet:327]: Migrating existing workflows to use read settings by default might be tedious, but when creating new repos it is quite straightforward. At some point we want to use read permissions by default, but its up to you how to do it right now.

Comment (MP91) [reply otterdog/eclipse-velocitas.jsonnet:327]: lets keep it synced over all repos for now and change all together later

Comment (netomi) [reply otterdog/eclipse-velocitas.jsonnet:327]: as you prefer

**Review by ariexi — APPROVED** [review]
LGTM

**Pull request merged**